### DISCUSSION TITLE
compile FE error: thrift failed to execute

### DISCUSSION THREAD

**Discussion by liukun4515** [apache/doris#15962] [A - General / Q&A]
ENV:
Mac
Error:

```
thrift failed to execute because: /Users/xxxx/.m2/repository/com/google/guava/guava/29.0-jre/guava-29.0-jre.jar was not a readable artifact -> [Help 1]
```

**Comment by liukun4515** [comment]
I just want to build FE code and test features

**Comment by yuanyuan8983** [comment]
Let's take a look at this problem. You can add me to WeChat:cyllyy810222

**Comment by linzch3** [comment]
maybe you should check this file: /Users/xxxx/.m2/repository/com/google/guava/guava/29.0-jre/guava-29.0-jre.jar

such as ls or view it's content to check if is it a corrutpted jar or anything else
